init.zsh

```diff
@@ -17,6 +17,45 @@ if ! autoload -Uz is-at-least || ! is-at-least "$min_zsh_version"; then
 fi
 unset min_zsh_version
 
+# zprezto convenience updater
+function zprezto-update {
+   function cannot-fast-forward {
+       local STATUS="$1"
+       [[ "${STATUS}" ]] && printf "%s\n" "${STATUS}"
```

Comment:
`[[ -n "${STATUS}" ]]` is the same thing as `[[ "${STATUS}" ]]`.

Should I explicitly add the -n? (this works without -n even in posix shell by the way)